src/torch/1-split-encoder-classifier/split-encoder-classifier.lua

```diff
@@ -69,17 +71,17 @@ end
 
 -- Test encoder reduction
 local fakeIm = torch.Tensor(3, 224, 224):cuda()
-local outputEnc = encoder:forward(fakeIm)
+outputEnc = encoder:forward(fakeIm)
```

Comment:
`outputEnc` is declared above, redeclaring variables is flagged as an error by luacheck